File: Bubble Sort/BubbleSortDart/BubbleSort.dart
List<int> bubbleSort(List<int> input) {
  
  List<int> numbers = input;
  int length = input.length;

  for (int k = 0; k < length; k++) {
    for(int i = 0; i < length - 1; i++) {
      if(numbers[i] > numbers[i + 1]) {
        int temp = numbers[i];
        numbers[i] = numbers[i + 1];
        numbers[i + 1] = temp;
      }
    }
  }

  return numbers;
}

main() {
  List<int> numbers = [45,88,0,-23,66,1];
  print(bubbleSort(numbers));
}

File: Bubble Sort/BubbleSortPHP/bubbleSort.php
<?php
// function for bubble sort
function bubblesort(&$Array, $n) 
{
  $temp;
  for($i=0; $i<$n; $i++) 
  {
    for($j=0; $j<$n-$i-1; $j++)
    {
      if($Array[$j]>$Array[$j+1])
      {
        $temp = $Array[$j];
        $Array[$j] = $Array[$j+1];
        $Array[$j+1] = $temp;
      }
    }
  }
}

// function to print array
function PrintArray($Array, $n) 
{ 
  for ($i = 0; $i < $n; $i++) 
    echo $Array[$i]." "; 
} 

// test the code
$MyArray = array(1, 10, 23, 50, 4, 9, 35);
$n = sizeof($MyArray); 
echo "Original Array\n";
PrintArray($MyArray, $n);

bubblesort($MyArray, $n);
echo "\nSorted Array\n";
PrintArray($MyArray, $n);
?>